lib/travis/api/v3/services/job/debug.rb

```diff
@@ -0,0 +1,26 @@
+module Travis::API::V3
+  class Services::Job::Debug < Service
+
+    def run
+      raise LoginRequired unless access_control.logged_in? or access_control.full_access?
+      raise NotFound      unless job = find(:job)
+      access_control.permissions(job).debug!
+
+      job.config.merge! debug_data
+      job.save!
+
+      query.restart(access_control.user)
+      accepted(job: job, state_change: :created)
+    end
+
+    def debug_data
```

Comment:
ahhhhh, I don't understand what you mean by that. If a user wants a debug session which doesn't print to the log how would he do that?